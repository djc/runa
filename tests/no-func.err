no-func.rns [2.5]: undefined name 'blah'
    blah()
    ^
