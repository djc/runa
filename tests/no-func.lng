def main(name: str, args: array[str]):
	blah()
