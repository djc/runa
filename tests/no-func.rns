def main(name: &str, args: &array[str]) -> i32:
	blah()
	return 0
