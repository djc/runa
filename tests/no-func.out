tests/no-func.lng [2.6]: not a function or method
    blah()
        ^
